steps/email.ts

```diff
@@ -0,0 +1,19 @@
+import { isAfter, parseISO, subMinutes } from 'date-fns'
+import { NotifyClient } from 'notifications-node-client'
+import { expect } from '@playwright/test'
+
+export const verifyEmailSent = async (subject: string, emailAddress: string) => {
```

Comment:
`emailAddress: TestOptions['user']['email']` might be possible; not sure if there's another type for the subject parameter